packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -71,22 +74,22 @@ watchEffect(() => {
   }
 })
 
-watch(() => isAlertDisplayed.value, (newVal) => {
+watch(() => isAlertDisplayed.value, async (newVal) => {
   if (!newVal) {
-    updateBannerState('dismissed')
+    await updateBannerState('dismissed')
   }
 })
 
-onMounted(() => {
-  updateBannerState('lastShown')
+onMounted(async () => {
+  await updateBannerState('lastShown')
```

Comment:
I don't think this `await` actually does anything - the hooks are async, `onMounted` will run the execution will go on without waiting. Was this just a style change?

Reply:
Mostly stylistic - I made the `updateBannerSync` function async, so flowed the `await` up through all usages just so the Promise-nature isn't lost. Probably a no-op for now, but if additional logic is added to `onMounted` in the future it will at least execute in order within the callback